### DISCUSSION TITLE
I am getting this message

### DISCUSSION THREAD

**Discussion by Juntropy** [Avnsx/fansly#160] [General]
OSError: dlopen(/Users/isername/opt/anaconda3/lib/python3.9/site-packages/pyexiv2/lib/libexiv2.dylib, 0x0006): Library not loaded: /usr/local/opt/gettext/lib/libintl.8.dylib
  Referenced from: /Users/username/opt/anaconda3/lib/python3.9/site-packages/pyexiv2/lib/libexiv2.dylib
  Reason: tried: '/usr/local/opt/gettext/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/usr/lib/libintl.8.dylib' (no such file)

**Comment by Avnsx** [comment]
Please open a issue ticket [here](https://github.com/LeoHsiao1/pyexiv2/issues/new), in which you further describe how exactly the error happend and a full traceback would be helpfull.

**Comment by greenwood172** [comment]
I ran into this issue / error as well  haven't had time to fully submit an issue. I believe it stems from (at least for me) being on an M1 mac and it seems like pyexiv2 doesn't have recent versions for arm64 architecture. See their packages here (which are x86_64 for macos) https://pypi.org/project/pyexiv2/2.8.2/#files
So, that itself results in an inability to find `pyexiv2>=2.8.2` although i'm able to then do a `pip install pyexiv2` which installs 2.5.x . However from that point, when I run the fansly downloader script, I get the aforementioned error.

Now obviously I understand that happens partly due to me using a version of pyexiv2 which is unsupported by the fansly downloader, but I would bet that there wouldn't be the same issues on a non ARM mac.

I do have an older mac that is x86_64 (that I don't use frequently) so I can try installing fansly-downloader on that computer and see how well it goes.
